md/database-configuration.md

```diff
@@ -171,12 +165,6 @@ It is assumed in the procedure that:
 
 #### SQL Server
```

Comment:
SQL Server section has no reason to exist here now.

Reply:
yes it does! It is the main section of the sub-sections below !

Reply:
Ouch. My bad